<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkModeClass</class>
 <widget class="QWidget" name="WorkModeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>900</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	background-color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>820</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>331</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 116, 174);
border-radius: 15px;   </string>
    </property>
    <property name="text">
     <string>作业选择</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_confirm">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>820</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>1021</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_01">
         <item>
          <widget class="QRadioButton" name="radioButton_func_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/接支接线路引线.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>接支接线路引线</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_02">
         <item>
          <widget class="QRadioButton" name="radioButton_func_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/断支接线路引线.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>断支接线路引线</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_03">
         <item alignment="Qt::AlignTop">
          <widget class="QRadioButton" name="radioButton_func_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/更换避雷器.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>更换避雷器</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButton_func_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/接跌落式熔断器上引线.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>接熔断器上引线</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioButton_func_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/断跌落式熔断器上引线.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>断熔断器上引线</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_06">
         <item>
          <widget class="QRadioButton" name="radioButton_func_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/更换跌落式熔断器.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>更换熔断器</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_07">
         <item>
          <widget class="QRadioButton" name="radioButton_func_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/装故障指示器.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>装故障指示器</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_08">
         <item>
          <widget class="QRadioButton" name="radioButton_func_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/拆故障指示器.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>拆故障指示器</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_09">
         <item>
          <widget class="QRadioButton" name="radioButton_func_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QRadioButton::indicator { 
	width: 200px;
	height: 200px;
}
/*单选框未选中样式*/
QRadioButton::indicator::unchecked {
	image: url(./picture/work_mode/除枝.png);
}
/*单选框选中样式*/
QRadioButton::indicator::checked { 
	image: url(./picture/work_mode/选中.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: rgb(0, 0, 0);
	font: 75 10pt &quot;微软雅黑&quot;;
}</string>
           </property>
           <property name="text">
            <string>除枝</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
